src/main/pages/che-7/end-user-guide/proc_defining-maven-repositories-in-the-settings-xml-file.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// using-maven-artifact-repositories
+
+[id="defining-repositories-in-settings-xml_{context}"]
+= Defining repositories in `settings.xml`
+
+To use your own artifact repositories at `example.server.org`, you must modify the  `settings.xml` file.
+Ensure, that the `settings.xml` is present in all the containers that use Maven tools, in particular the Maven container and the Java plugin container.
+
+By default, `settings.xml` is located at the user home directory, but since user home directory is not on a persistent volume, each time you restart workplace, you must re-create the file.
+
+To make the file persist between the restarts of the workspace, create a directory `/projects/maven/.m2` and move the `settings.xml` file in it. Then set the value of the `user.home` Java system property to `/projects/maven` in all your Maven containers.
```

Comment:
This only makes sense as a workaround for issue https://github.com/eclipse/che/issues/13318. If we don't mention why we're doing this workaround, folks will be confused.